### PR TITLE
Foster filtering resolves #314

### LINKED ISSUE #314: Foster Filtering
From the Foster's Tab of the Staff Directory (aka User Index)

A user should be able to filter the list of fosters by any or none following criteria:

state  :string   (Type in the state abbrv.)

available_to_foster    :boolean
agreement_id           :integer   (if present or not)
house_type             :string(40)   (own/rent)
breed_restriction      :boolean
weight_restriction     :boolean
has_own_dogs           :boolean
has_own_cats           :boolean
children_under_five    :boolean
has_fenced_yard        :boolean
can_foster_puppies     :boolean
parvo_house            :boolean

### PR DESCRIPTION
**Pull request by TheDudeWithTheThing**
* Filterable concern
* Many new scopes for user
* active no longer in all scopes
* Disable rollbar in non production environments

### CODE REVIEW

**Comment by TheDudeWithTheThing** [comment]
I hope they aren't using this on mobile!

operation_paws_for_homes___staff_directory

**Comment by docvego** [comment]
hahaha, I doubt very often but it looks like it would still at least work in a pinch.  We'll get thing extra pretty once we do the Bootstrap rebuild

File: app/views/users/_foster_filters.html.erb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  <input type="hidden" name="foster" value="true"/>
+
+  <label class="pull-right">
+    Available
+    <%= select_tag "available_to_foster", options_for_select(@options, params[:available_to_foster]), class: 'input-mini' %>
+  </label>
+
+  <label class="pull-right">
+    Waiver
+    <%= select_tag "available_to_foster", options_for_select(@options, params[:available_to_foster]), class: 'input-mini' %>
```
Comment (docvego) [inline app/views/users/_foster_filters.html.erb]: Duplicate tag and options params :point_up_2:

Comment (TheDudeWithTheThing) [inline app/views/users/_foster_filters.html.erb]: Oh yeah, meant to ask you where the waiver field is, is it a new one?

Comment (docvego) [inline app/views/users/_foster_filters.html.erb]: The waiver is an Attachment object to User.  Basically an uploaded pdf that they signed.

Comment (TheDudeWithTheThing) [inline app/views/users/_foster_filters.html.erb]: what's that mean in db terms?  where attachments.updated_by_user_id = user.id ?

Comment (docvego) [inline app/views/users/_foster_filters.html.erb]: You might just be able to hit `agreement`   See https://github.com/ophrescue/RescueRails/blob/master/app/views/users/show.html.erb#L109

**Comment by TheDudeWithTheThing** [comment]
@docvego should be good, test her out!

**Comment by docvego** [comment]
Looks Good

**Pull request merged**